apps/wakunode2/external_config.nim

```diff
@@ -552,4 +552,19 @@ proc load*(T: type WakuNodeConf, version=""): ConfResult[T] =
   except CatchableError:
     err(getCurrentExceptionMsg())
 
+proc defaultTestWakuNodeConf*(): WakuNodeConf =
+  WakuNodeConf(
+    tcpPort: Port(60000),
+    websocketPort: Port(8000),
+    listenAddress: ValidIpAddress.init("0.0.0.0"),
+    rpcAddress: ValidIpAddress.init("127.0.0.1"),
+    restAddress: ValidIpAddress.init("127.0.0.1"),
+    metricsServerAddress: ValidIpAddress.init("127.0.0.1"),
+    dnsAddrsNameServers: @[ValidIpAddress.init("1.1.1.1"), ValidIpAddress.init("1.0.0.1")],
+    nat: "any",
+    maxConnections: 50,
+    topics: @["/waku/2/default-waku/proto"],
+    relay: true
+  )
```

Comment:
Why has this been moved from `test_app.nim`? I wouldn't move any test code into "production" code.